db/services.php

```diff
@@ -0,0 +1,181 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * mod_kanban service definition.
+ *
+ * @package    mod_kanban
+ * @copyright  2023 ISB Bayern
+ * @author     Stefan Hanauska
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+defined('MOODLE_INTERNAL') || die;
+
+$functions = [
+    'mod_kanban_get_kanban_content_init' => [
+        'classname'   => 'mod_kanban\external\get_kanban_content',
+        'methodname'  => 'get_kanban_content_init',
+        'description' => 'Retrieves the content of the kanban board',
+        'type'        => 'read',
+        'ajax' => true,
+        'capabilities' => 'mod/kanban:view',
+    ],
+    'mod_kanban_get_kanban_content_update' => [
+        'classname'   => 'mod_kanban\external\get_kanban_content',
+        'methodname'  => 'get_kanban_content_update',
+        'description' => 'Retrieves the content of the kanban board',
```

Comment:
better description: "Retrieves only updated content of the kanban board since last timestamp"